src/main/java/mate/academy/internetshop/controllers/ProductAddToCartController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.service.ProductService;
+import mate.academy.internetshop.service.ShoppingCartService;
+
+@WebServlet("/products/add")
+public class ProductAddToCartController extends HttpServlet {
```

Comment:
```suggestion
public class AddProductToCartController extends HttpServlet {
```

Reply:
I decided to name it as Product so that it stays on one alphabetical line with all Products, can I leave it like this? Or it's better to change? It's just easier to search when everything in one line

Reply:
Done